app/src/main/java/com/example/dashcarr/presentation/tabs/social/selectMessage/SelectMessageFragment.kt

```diff
@@ -108,6 +138,47 @@ class SelectMessageFragment : BaseFragment<FragmentSelectMessageBinding>(
             sendTextMessageBelowAndroid12(destinationNumer, message)
         }
 
+
+    }
+
+    fun sendEmail(destinationEmail: String, message: String) {
+        val stringSenderEmail = "dashcarr.business@gmail.com"
+        val stringRecieverMail = destinationEmail
+        val stringPasswordSenderEmail = BuildConfig.DASHCARR_SMTP_PASSWORD
+
+
+        appExecutors.diskIO().execute {
+            val props = System.getProperties()
+            props.put("mail.smtp.host", "smtp.gmail.com")
+            props.put("mail.smtp.socketFactory.port", "465")
+            props.put(
+                "mail.smtp.socketFactory.class",
+                "javax.net.ssl.SSLSocketFactory"
+            )
+            props.put("mail.smtp.auth", "true")
+            props.put("mail.smtp.port", "465")
+            val session = Session.getInstance(props,
+                object : javax.mail.Authenticator() {
+                    override fun getPasswordAuthentication(): PasswordAuthentication {
+                        return PasswordAuthentication(stringSenderEmail, stringPasswordSenderEmail)
+                    }
+                }
+            )
+            try {
+                val mm = MimeMessage(session)
+                mm.setFrom(InternetAddress(stringSenderEmail))
+                mm.addRecipient(
+                    Message.RecipientType.TO,
+                    InternetAddress(stringRecieverMail)
+                )
+                mm.subject = "Message from: ${userViewModel.getUser()?.email}"
+                mm.setText("Hi!\n\nYou have gotten a message from: ${userViewModel.getUser()?.email}\nMessage:\n\n$message")
+
+                Transport.send(mm)
```

Comment:
![Skjermbilde 2023-12-02 kl  00 36 31](https://github.com/StianJohnsen/dte2503_team_3_repo/assets/97880997/8f8a53db-fc75-431a-94f6-22731cf5349b)

crush on sending email
might be an issue with the JavaMail library, probably missing dependency at least

Reply:
btw don't forget rebase as well, it could be better fix the bug having updated version from master